noah/src/main/resources/crawler/health_map_crawler.py

```diff
@@ -0,0 +1,95 @@
+import urllib2
+import urllib
+import datetime
+import io
+import os
+import json
+import sys
+ 
+
+#get health map response object
+def get_health_map(url, headers):
+    request = urllib2.Request(url, headers = headers)
+    response = urllib2.urlopen(request)
+    return response
+
+# create the parameters, needs start_date and end_date in "mm/dd/yyyy" format
+def generate_health_map_query(start_date, end_date):
+    parameter_map = {"category":["1","2","29"],"diseases":["282"], "heatscore":"1", "partner":"hm"}
+    parameter_map['sdate'] = start_date
+    parameter_map['edate'] = end_date
+    return parameter_map
+    
+    
+def getUrl(alertid, trto, trfr, pid):
+    query = {}
+    query['trto'] = trto
+    query['trfr'] = trfr
+    url = "http://www.healthmap.org/ai.php?"+alertid+"&"+urllib.urlencode(query)+"&"+"pid"+pid
+    return url
+
+def storeHtml(response, fname, headers):
+    request = urllib2.Request(url, headers = headers)
+    response = urllib2.urlopen(request)
+    with io.open(fname+".html", "w", encoding="utf-8") as writeFile:
+        writeFile.write(response.read().decode("utf-8"))
+        writeFile.write(u"\u000A")
```

Comment:
That is the unicode character \n i.e; new line. It is added at the end after writing the whole file. I don't think it is needed. I deleted it.

Done.